### PR TITLE
Added (incomplete) blueprints.csv

### PR DESCRIPTION
**Pull request by barrywimlett**
Added (incomplete) blueprints.csv

### CODE REVIEW

File: blueprints.csv [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+128673820,ShieldGenerator_Kinetic
+128673825,ShieldGenerator_Optimised
+128673835,ShieldGenerator_Reinforced
+128673842,ShieldGenerator_Thermic
+128682083,Weapon_Sturdy
+128731526,Misc_LightWeight
+128739090,Weapon_Efficient
+128740134,Sensor_LongRange
+128740138,Sensor_WideAngle
+128740153,Sensor_Expanded
```
Comment (Tkael) [inline blueprints.csv:30]: My understanding is that there is a separate blueprint for each grade and that each has its own ID. Linking EDDI's definitions (which may also be imperfect) for reference: https://github.com/EDCD/EDDI/blob/develop/DataDefinitions/Blueprint.cs

Comment (barrywimlett) [reply blueprints.csv:30]: thanks for the observation, I hadn't noticed each grade had its own id

*grumbles incohernetly*

Comment (barrywimlett) [reply blueprints.csv:30]: ugh it gets worse 

not only does each blueprint grade have its own id, but it also varies depending on the module you apply it to, so weapon-efficient has different IDs for multicannon,beamlaser etc.

**Comment by robbyxp1** [comment]
So this is not complete, please resubmit if you want this in

**Comment by barrywimlett** [comment]
yeah its a WIP sorry I am still working on it , yeah I can always update the file and raise a pull-request - thanks to those who commented - it helped even if it wasn't what I wanted

**Pull request closed without merging**